next.config.js

```diff
@@ -14,7 +14,16 @@ const nextConfig = {
     appDir: true,
   },
   reactStrictMode: true,
-  webpack: (config) => {
+  webpack(config) {
+    config.resolve.fallback = {
+      fs: false,
+      net: false,
+      tls: false,
+      'react-native': false,
+    }
+    config.externals.push('pino-pretty', 'lokijs', 'encoding')
+    config.optimization.moduleIds = 'named'
+    config.optimization.runtimeChunk = 'single'
```

Comment:
next time avoid config changes unrelated to the PR. this PR needed to urgently get merged and this config changes makes the PR with more chances of introducing unexpected bugs

Reply:
Yes ser. It was some lines to stop some weird terminal logs caused by pino pretty :))